ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.12.yaml

```diff
@@ -58,7 +58,7 @@ tests:
   interval: 960h
   steps:
     cluster_profile: gcp-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi-sdn
+    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-sdn2ovn
```

Comment:
If we want to check `gcp-upi-ovn-sdn2ovn` for both 4.11 and 4.12, then I think here should be `upi` instead of `ipi`.

Reply:
guess this one is mistake updated